### ISSUE TITLE
Bahasa Malaysia

### ISSUE THREAD

**Issue by murugappanrm** [squidfunk/mkdocs-material#3447]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translation

```jinja
{% macro t(key) %}{{ {
  "language": "ms",
  "direction": "ltr",
  "clipboard.copy": "Salin ke papan keratan",
  "clipboard.copied": "Disalin ke papan keratan",
  "edit.link.title": "Edit halaman ini",
  "footer.previous": "Sebelumnya",
  "footer.next" : "Seterusnya",
  "footer.title": "Pengaki",
  "header.title": "Pengepala",
  "meta.comments": "Komen",
  "meta.source": "Sumber",
  "nav.title": "Navigasi",
  " search. config.lang": "ms",
"search. config.pipeline":"trimmer,  stopWordFilter",
  " search. config.separator": "[\s\-]+",
  "search.placeholder": "Cari",
  "search.reset": "Padam",
  "search.result.initializer": "Siap carian",
  "search.result.placeholder": "Taip untuk mula mencari",
  "search.result.none": "Tiada dokumen yang sepadan",
  "search.result.one": "1 dokumen yang sepadan",
  "search.result.other": "# dokumen yang sepadan",
  "search.result.more.one": "1 lagi di halaman ini",
  "search.result.more.other": "# lagi di halaman ini",
  "search.result.term.missing": "Hilang",
  "select.language.title": "Pilih bahasa",
  "select.version.title": "Pilih versi",
  "skip.link.title": "Langkau tajuk talian",
  "source.link.title": "tajuk talian asal",
  "source.file.date.updated": "Tarikh fil dikemas kini",
  "source.file.date.created": "tarikh fil asal dicipta",
  "tabs.title": "Tab",
  "toc.title": "Jadual kandungan",
  "top.title": "Kembali ke atas"

}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Added in cffb40343

**Comment by squidfunk** [comment]
Released as part of 8.1.5 – this the 50th language added to Material for MkDocs 🎉
